mdaviz/mainwindow.py

```diff
@@ -229,7 +226,7 @@ def get_all_subfolders(
                 settings.setKey(DIR_SETTINGS_KEY, ",".join(recent_dirs))
 
             else:
-                comment = f"{str(folder_path)!r} - invalid path."
+                comment = f"\n{str(folder_path)!r} - invalid path."
```

Comment:
Can this be combined with the `setStatus(comment)` call below?  Only used once here.

Reply:
Yep, that's legacy stuff.